cves/CVE-2014-0098.yml

```diff
@@ -118,7 +125,7 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
+  answer: The vulnerability was found by Rainer Canavan internally.
   date:
```

Comment:
Was there a date that the vulnerability was found?

Reply:
Couldn't find any, it was just referenced in the commits.